packages/flutter_tools/gradle/src/main/kotlin/DependencyVersionChecker.kt

```diff
@@ -105,6 +131,35 @@ object DependencyVersionChecker {
         // KGP is not required, so don't log any warning if we can't find the version.
     }
 
+    private fun checkMinSdkForAndroidComponents(project: Project) {
```

Comment:
Naming nit: I think this should be called something like `configureMinSdkCheck` since that is what this method does and the actual check is in `checkMinSdkVersion` which feels appropriately named.